build-tools/automation/azure-pipelines.yml

```diff
@@ -0,0 +1,57 @@
+# Xamarin.Android Pipeline
+
+trigger:
+  - master
+  - d16*
+
+stages:
+- stage: Build
+  jobs:
+  - job: Mac
+    pool: Hosted Mac Internal
+    timeoutInMinutes: 240
+    cancelTimeoutInMinutes: 5
+    variables:
+      configuration: Release
+      msbuildArgs: /p:AutoProvision=True /p:AutoProvisionUsesSudo=True /p:IgnoreMaxMonoVersion=False
+      JavaSdkDirectory: /Library/Java/JavaVirtualMachines/jdk1.8.0_144.jdk/Contents/Home/
+      JAVA_HOME: $(JavaSdkDirectory)
+    steps:
+    - script: make prepare-external-git-dependencies CONFIGURATION=$(configuration)
+      displayName: make prepare-commercial
+      condition: eq(variables['XA.Commercial.Build'], 'true')
+      env:
+        GH_AUTH_SECRET: $(Github.Token)
+    - task: provisionator@2
+      condition: and(succeeded(), eq(variables['XA.Commercial.Build'], 'true'))
+      inputs:
+        provisionator_uri: $(provisionator-uri)
+        github_token: $(GitHub.Token)
+        provisioning_script: $(System.DefaultWorkingDirectory)/external/monodroid/build-tools/provisionator/profile.csx
+        provisioning_extra_args: -vv
+    - script: make prepare-deps V=1 CONFIGURATION=$(configuration) MSBUILD_ARGS="$(msbuildArgs)"
+      displayName: make prepare-deps
+    - script: make prepare jenkins V=1 CONFIGURATION=$(configuration) MSBUILD_ARGS="$(msbuildArgs)"
+      displayName: make prepare jenkins
+    - script: make create-installers V=1 CONFIGURATION=$(configuration)
+      displayName: make create-installers
+    - script: |
+       mkdir -p $(System.DefaultWorkingDirectory)/unsigned-installers
```

Comment:
Ditto here: Unix shell commands.  We could instead create a "global" MSBuild target in `Before.Xamarin.Android.sln.targets` which performs these "same" actions.

Reply:
Marking as resolved as per the comment above, I think it's ok to retain a few unix commands related to artifact publishing in this macOS-only pipeline.